src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,22 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{comment.name}</strong>
+
+      {' by '}
+
+      <a
+        className="CommentInfo__email"
+        href={`mailto:${comment.email}`}
```

Comment:
If the email is undefined, then we will have an error when clicking, try to predict it